site/gatsby-site/src/templates/landingPage.js

```diff
@@ -41,12 +41,12 @@ const LandingPage = (props) => {
     const report = sortedReports.map((report) => report)[0];
 
     if (report.language !== language) {
-      const translation = data[`latestReports_${language}`].edges.find(
+      const translation = data[`latestReports_${language}`]?.edges.find(
         (translation) => translation.node.report_number === report.report_number
       );
 
-      report.title = translation.node.title;
-      report.text = translation.node.text;
+      report.title = translation?.node?.title || '';
+      report.text = translation?.node?.text || '';
```

Comment:
Please apply this [fix](https://github.com/responsible-ai-collaborative/aiid/blob/308cbfe72edfe86389222105af0cfd283e25e308/site/gatsby-site/src/templates/landingPage.js#L56C7-L61C8) instead